bagit.py

```diff
@@ -388,7 +392,7 @@ def _load_manifests(self):
             alg = os.path.basename(manifest_file).replace(search, "").replace(".txt", "")
             self.algs.append(alg)
 
-            with open(manifest_file, 'r') as manifest_file:
+            with open_text_file(manifest_file, 'r', encoding='utf-8-sig') as manifest_file:
```

Comment:
According to the [bagit spec](https://tools.ietf.org/html/draft-kunze-bagit-13#section-2.1.1), we would need to open this not as utf-8 but as whatever is specified in the bagit.txt file.